testing/functional_tests/test_automate_training.py

```diff
@@ -28,6 +29,18 @@ def test_automate_training(download_functional_test_files, script_runner):
     assert os.path.exists(os.path.join(__output_dir__, 'temporary_results.csv'))
     assert os.path.exists(os.path.join(__output_dir__, 'average_eval.csv'))
 
+    # check sha256 is recorded in config_file.json
+    initial_config = imed_config_manager.ConfigurationManager(file_config).get_config()
+    result = []
+    name = "config_file.json"
+    for root, dirs, files in os.walk(os.path.dirname(initial_config["path_output"])):
+        if name in files:
+            result.append(os.path.join(root, name))
+    assert result != []
+    for generated_config in result:
+        config = imed_config_manager.ConfigurationManager(generated_config).get_config()
+        assert 'training_sha256' in config
+
```

Comment:
This is quite similar to the code block below, do you think there is need/chance for code reuse? Do you foresee this pattern being used more often in future implementations or is this going to be only relevant in the automated_training section?

Reply:
I put it in utils instead